src/data.js

```diff
@@ -0,0 +1,199 @@
+import React from "react";
+import { Fragment } from "react";
+
+export default {
+  site: {
+    title: "Le Petit Bloc",
+    baseline: "Build a decentralised, trustable and reliable world",
+  },
+
+  social: [
+    {
+      icon: "fab fa-twitter",
+      title: "Link to our Twitter account",
+      href: "https://twitter.com/lepetitbloc"
+    },
+    {
+      icon: "fab fa-github",
+      title: "Link to our Github account",
+      href: "https://github.com/lepetitbloc"
+    },
+    {
+      icon: "fab fa-docker",
+      title: "Link to our Docker account",
+      href: "https://hub.docker.com/u/lepetitbloc/"
+    },
+    {
+      icon: "fab fa-discord",
+      title: "Link to our Discord account",
+      href: "https://discordapp.com/invite/PhdnNnf"
+    },
+    {
+      icon: "fas fa-envelope-open",
+      title: "Contact or Hire us",
+      href: "mailto:bonjour@lepetitbloc.net?subject=Bonjour%20Le%20Petit%20Bloc"
+    }
+  ],
+
+  timecard: [
+    {
+      badge: "fas fa-link",
+      color: "warning",
+      title: "New website",
+      content:
```

Comment:
If we need formatting, I'd rather see **markdown** here instead of jsx....
Let me dig some references for this...